Refresh Visual Studio Edge debugging docs

captainbrosset opened the pull request:
The project template to be used in VS 2019 changed so the documentation wasn't correct anymore. The wording had to be changed and so did the screenshot.

The UI of the react.js webapp also changed, so the screenshots had to beupdated too.

I also changed some of the wording as per the MWSG.

Fixes #1350

opbld32 commented:
Docs Build status updates of commit _[79b901a](https://github.com/MicrosoftDocs/edge-developer/commits/79b901ab1137a6ebe4b4b9cd02ac5349e2eb6579)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-edge/visual-studio/index.md](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |
[microsoft-edge/visual-studio/media/create-new-project.png](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/media/create-new-project.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |
[microsoft-edge/visual-studio/media/edge-counter.png](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/media/edge-counter.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |
[microsoft-edge/visual-studio/media/edge-launch.png](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/media/edge-launch.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C7%5C27%5Cdb9f4228-bcf7-7df9-527f-b7c9a032ef40%5CPullRequest%5C202107270933437496-1360%5Cworkflow_report.html?skoid=5f458e1a-fa67-4ae6-a972-54ea42ec670e&sktid=975f013f-7f24-47e8-a7d3-abc4752bf346&skt=2021-07-26T06%3A14%3A02Z&ske=2021-08-02T06%3A19%3A02Z&sks=b&skv=2020-04-08&sv=2020-04-08&st=2021-07-27T09%3A29%3A09Z&se=2021-08-27T09%3A34%3A09Z&sr=b&sp=r&sig=jHhphS9PTi8XqhCLaAg%2FMBVWsiNNNLGLYy94BW63yA0%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/db9f4228-bcf7-7df9-527f-b7c9a032ef40?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld31 commented:
Docs Build status updates of commit _[ffa6cbd](https://github.com/MicrosoftDocs/edge-developer/commits/ffa6cbd59b78bbe20014d55dcf37ff3a9086bf71)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-edge/visual-studio/index.md](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |
[microsoft-edge/visual-studio/media/create-new-project.png](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/media/create-new-project.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |
[microsoft-edge/visual-studio/media/edge-counter.png](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/media/edge-counter.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |
[microsoft-edge/visual-studio/media/edge-launch.png](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/media/edge-launch.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C7%5C27%5Cdb9f4228-bcf7-7df9-527f-b7c9a032ef40%5CPullRequest%5C202107270935041801-1360%5Cworkflow_report.html?skoid=5f458e1a-fa67-4ae6-a972-54ea42ec670e&sktid=975f013f-7f24-47e8-a7d3-abc4752bf346&skt=2021-07-26T06%3A14%3A02Z&ske=2021-08-02T06%3A19%3A02Z&sks=b&skv=2020-04-08&sv=2020-04-08&st=2021-07-27T09%3A30%3A30Z&se=2021-08-27T09%3A35%3A30Z&sr=b&sp=r&sig=g5QgKQxAGZr7Qyz6Vjij9RrU%2F7wLI8A14B6eqT6Cnag%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/db9f4228-bcf7-7df9-527f-b7c9a032ef40?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

mikehoffms commented on microsoft-edge/visual-studio/index.md:
```diff
@@ -34,52 +34,49 @@ The simplified workflow allows you to debug JavaScript that run in Microsoft Edg
 ### Launch Microsoft Edge from Visual Studio  
 
-After you create your project, open `ClientApp/src/components/Counter.js`.  Now, to use Visual Studio to debug JavaScript, choose the dropdown next to the green **Play** button and **IIS Express**.  
+After you create your project, open `ClientApp/src/components/Counter.js`.  Now, to use Visual Studio to debug JavaScript, select the dropdown next to the green **Play** button and **IIS Express**.  
```
```suggestion
1.  After you create your project, open `ClientApp/src/components/Counter.js`.

1.  Now, to use Visual Studio to debug JavaScript, select the dropdown next to the green **Play** button and **IIS Express**.  
```
For closely related (conceptually grouped) user-actions that feel like a single conceptual step, group into a single numbered list item.  Avoid ultra-granular, fragmented numbered steps.
blank line spreads numbered steps apart, for readability.  
For the image block below, prefix spaces to move it right so it starts in column 4.  I think most Markdown, including this build system, treats column 4 as meaningful/significant; have to check the Doc Contributor Guide later.

mikehoffms commented on microsoft-edge/visual-studio/index.md:
```diff
@@ -34,52 +34,49 @@ The simplified workflow allows you to debug JavaScript that run in Microsoft Edg
-After you create your project, open `ClientApp/src/components/Counter.js`.  Now, to use Visual Studio to debug JavaScript, choose the dropdown next to the green **Play** button and **IIS Express**.  
+After you create your project, open `ClientApp/src/components/Counter.js`.  Now, to use Visual Studio to debug JavaScript, select the dropdown next to the green **Play** button and **IIS Express**.  
 
 :::image type="complex" source="./media/vs-dropdown.png" alt-text="The dropdown next to the green Play button and IIS Express" lightbox="./media/vs-dropdown.png":::
    The dropdown next to the green **Play** button and **IIS Express**  
 :::image-end:::  
```
```suggestion
    :::image type="complex" source="./media/vs-dropdown.png" alt-text="The dropdown next to the green Play button and IIS Express" lightbox="./media/vs-dropdown.png":::
       The dropdown next to the green **Play** button and **IIS Express**  
    :::image-end:::  
```
Start in col 4(?) to indent image and maybe to make the numbered steps below auto-increment correctly.

mikehoffms commented on microsoft-edge/visual-studio/index.md:
```diff
@@ -34,52 +34,49 @@ The simplified workflow allows you to debug JavaScript that run in Microsoft Edg
 ### Debug JavaScript running in Microsoft Edge  
 
-Switch back to Visual Studio.  In `Counter.js`, to set a breakpoint on Line 13, choose the gutter next to the line.  
+Switch back to Visual Studio.  In `Counter.js`, to set a breakpoint on Line 13, select the gutter next to the line.  
```
Depending on how thorough of an update this PR aims for, test the steps, make sure it's still Line 13.  Testing takes a long time, but can yield corrections.  But consider/balance/weigh the opportunity cost of such in-depth testing of steps.

captainbrosset replied:
Yep, I did test locally, which is how I was able to refresh a couple of the screenshots. The line is still 13, so this is correct.

mikehoffms commented on microsoft-edge/visual-studio/index.md:
```diff
@@ -115,13 +112,13 @@ From the **Attach to Process** dialog, set **Connection type** to **Chrome devto
-Choose **Select...** > the checkbox next to **JavaScript (Microsoft Edge – Chromium)**.  To add tabs, navigate to new tabs, and close tabs and display the changes reflected in the **Attach to Process** dialog, choose the **Refresh** button.  Choose the tab you want to debug and choose **Attach**.  
+Select **Select** > the checkbox next to **JavaScript (Microsoft Edge – Chromium)**.  To add tabs, navigate to new tabs, and close tabs and display the changes reflected in the **Attach to Process** dialog, select the **Refresh** button.  Choose the tab you want to debug and select **Attach**.  
 
 The Visual Studio debugger is now attached to Microsoft Edge.  You may pause the running of JavaScript, set breakpoints, and review `console.log()` statements directly in the Debug Output window in Visual Studio.  
```
Global: check 'can' and 'may'.  Change 'may' to 'can' in most places.  Per MWSG's [can, may](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=33430) article: "When ability is what you need to express, it's OK to use can to describe actions or tasks that the reader or program is able to do. Use might to express possibility. **Don't use may**, which might be interpreted as providing permission."

mikehoffms replied:
```suggestion
The Visual Studio debugger is now attached to Microsoft Edge.  You can pause the running of JavaScript, set breakpoints, and review `console.log()` statements directly in the Debug Output window in Visual Studio.  
```
I'd keep this left-justified.  Optionally shift w/ leading spaces to start in column 4(?) if it looks more consistent.

mikehoffms commented on microsoft-edge/visual-studio/index.md:
```diff
@@ -34,52 +34,49 @@ The simplified workflow allows you to debug JavaScript that run in Microsoft Edg
 :::image-end:::  
 
-In the same dropdown, choose **Web Browser** > the preview channel of Microsoft Edge that you want Visual Studio to launch, such as Microsoft Edge Canary, Dev, or Beta.  If you are not already using one of the Microsoft Edge preview channels, navigate to [Download Microsoft Edge Insider Channels][MicrosoftedgeinsiderDownload] to download one.  
+In the same dropdown, select **Web Browser** > the preview channel of Microsoft Edge that you want Visual Studio to launch, such as Microsoft Edge Canary, Dev, or Beta.  If you are not already using one of the Microsoft Edge preview channels, navigate to [Download Microsoft Edge Insider Channels][MicrosoftedgeinsiderDownload] to download one.  
```
Elsewhere, change:
you may step through the script line-by-line
to:
you can step through the script line-by-line

mikehoffms commented on microsoft-edge/visual-studio/index.md:
```diff
@@ -95,18 +92,18 @@ The example was just a minor demonstration of the functionality available in Vis
+   Select **Attach to Process** in Visual Studio  
 :::image-end:::  
 
 From the **Attach to Process** dialog, set **Connection type** to **Chrome devtools protocol websocket (no authentication)**.  In the **Connecting target** textbox, type `http://localhost:9222/` and select `Enter`.  Review the list of open tabs you have in Microsoft Edge listed out in the **Attach to Process** dialog.  
```
```suggestion
1.  In the **Attach to Process** dialog, set **Connection type** to **Chrome devtools protocol websocket (no authentication)**.  
1.  In the **Connecting target** textbox, type `http://localhost:9222/` and select `Enter`.  In the **Attach to Process** dialog, review the list of tabs that are open in Microsoft Edge.  
```
Break into 2-3 cohesive steps.  Reading this step out of context, it's unclear how many dialog boxes are involved.  Each dialog would be a separate step.

mikehoffms commented on microsoft-edge/visual-studio/index.md:
```diff
@@ -34,52 +34,49 @@ The simplified workflow allows you to debug JavaScript that run in Microsoft Edg
 :::image-end:::  
 
-Now switch back to the instance of Microsoft Edge that Visual Studio launched.  Choose on **Counter** in the NavMenu on the left of the webpage.  Now choose **Increment**.  
+Now switch back to the instance of Microsoft Edge that Visual Studio launched.  Select **Counter** in the navigation menu on the left of the webpage.  Now select **Increment**.  
```
```suggestion
1.  Switch back to the instance of Microsoft Edge that Visual Studio launched.

1.  Select **Counter** in the navigation menu at the top of the webpage, and then select the **Increment** button.
```
Check the location of "Counter" UI, in context.    Looks like it's in the upper right in the screenshot.
Style guide says "don't state the UI control type, unless it helps clarity".  My retort is, it _always_ helps clarity, esp given the Accessibility rule, "don't assume the screenshots are visible to the audience".  I omit "the ... button" only in the easy case of the OK button.

mikehoffms commented on microsoft-edge/visual-studio/index.md:
```diff
@@ -34,52 +34,49 @@ The simplified workflow allows you to debug JavaScript that run in Microsoft Edg
 
-After you create your project, open `ClientApp/src/components/Counter.js`.  Now, to use Visual Studio to debug JavaScript, choose the dropdown next to the green **Play** button and **IIS Express**.  
+After you create your project, open `ClientApp/src/components/Counter.js`.  Now, to use Visual Studio to debug JavaScript, select the dropdown next to the green **Play** button and **IIS Express**.  
 
```
near top of article, change:
Visual Studio completes the following workflow using a single button.
to:
Visual Studio completes the entire following workflow when you select a single button.
(which button? does the surrounding text fill-in a mental picture adequately to follow the meaning here?  i'd try to slip-in a hint of what we have in mind)

captainbrosset replied:
This section wasn't very clear, I rephrased it.

mikehoffms commented on microsoft-edge/visual-studio/index.md:
```diff
@@ -34,52 +34,49 @@ The simplified workflow allows you to debug JavaScript that run in Microsoft Edg
+This **React.js** template specifies how to integrate React.js with an ASP.NET Core app.  
 
-Provide a **Project name** for your new project and choose **Create**.  For the purposes of the example, choose **React.js** as the template, and choose **Create**.  The **React.js** template specifies how to integrate React.js with an ASP.NET Core app.  
+:::image type="complex" source="./media/create-new-project.png" alt-text="Create a new ASP.NET Core Web app with React.js" lightbox="./media/create-new-project.png":::
```
If not too much trouble/time, in screenshots, change yellow box to red.  Usually 3px not 5px, but that width could be hard to change.  
To change yellow to red in Paint:
1. Shift+click an image link in the .md file in VS Code.
2. Use a menu item or Sh+Alt+R to open Explorer at the dir.
3. Right-click an image file, Edit, to open in Paint.
4. Select or enter a red color.  Or open an image with red box and use Dropper tool to sample it.
4. Select Paint Bucket tool, click a yellow rectangle.  Save.

captainbrosset commented on microsoft-edge/visual-studio/index.md:
```diff
@@ -115,13 +112,13 @@ From the **Attach to Process** dialog, set **Connection type** to **Chrome devto
 :::image-end:::  
 
-Choose **Select...** > the checkbox next to **JavaScript (Microsoft Edge – Chromium)**.  To add tabs, navigate to new tabs, and close tabs and display the changes reflected in the **Attach to Process** dialog, choose the **Refresh** button.  Choose the tab you want to debug and choose **Attach**.  
+Select **Select** > the checkbox next to **JavaScript (Microsoft Edge – Chromium)**.  To add tabs, navigate to new tabs, and close tabs and display the changes reflected in the **Attach to Process** dialog, select the **Refresh** button.  Choose the tab you want to debug and select **Attach**.  
```
Thanks for catching this. I was mostly focusing on the first part of the page (`Debug JavaScript running in Microsoft Edge`) and didn't realize that (`Attach to Microsoft Edge`) was incorrect. Turns out this whole section was very confusing as it was talking about VS and Edge at the same time but didn't make it clear. I've tested the workflow again myself and will update accordingly.

captainbrosset commented on microsoft-edge/visual-studio/index.md:
```diff
@@ -95,18 +92,18 @@ The example was just a minor demonstration of the functionality available in Vis
 ## Attach to Microsoft Edge  
 
-Previously, you had to launch Microsoft Edge from Visual Studio.  Now, you are able to attach the Visual Studio debugger to an already running instance of Microsoft Edge.  
+Previously, you launched Microsoft Edge from Visual Studio.  You can also attach the Visual Studio debugger to an already running instance of Microsoft Edge.  
```
Yeah, 2 alternative approaches that can't be used at the same time. I'll update accordingly.

opbld30 commented:
Docs Build status updates of commit _[7135a66](https://github.com/MicrosoftDocs/edge-developer/commits/7135a66f2039b7704df9f76b30aa26257bcf639a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-edge/visual-studio/index.md](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |
[microsoft-edge/visual-studio/media/attach-to-process.PNG](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/media/attach-to-process.PNG) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |
[microsoft-edge/visual-studio/media/create-new-project.png](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/media/create-new-project.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |
[microsoft-edge/visual-studio/media/edge-counter.png](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/media/edge-counter.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |
[microsoft-edge/visual-studio/media/edge-launch.png](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/media/edge-launch.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |
[microsoft-edge/visual-studio/media/enable-script-debugging.png](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/media/enable-script-debugging.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |
[microsoft-edge/visual-studio/media/feedback-icon.png](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/media/feedback-icon.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |
[microsoft-edge/visual-studio/media/set-breakpoint.png](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/media/set-breakpoint.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |
[microsoft-edge/visual-studio/media/set-web-browser.png](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/media/set-web-browser.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |
[microsoft-edge/visual-studio/media/vs-dropdown.png](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/media/vs-dropdown.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |
[microsoft-edge/visual-studio/media/vs.msft.png](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/media/vs.msft.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/media/vs.msft.png?branch=pr-en-us-1360) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C7%5C28%5Cdb9f4228-bcf7-7df9-527f-b7c9a032ef40%5CPullRequest%5C202107280928312087-1360%5Cworkflow_report.html?skoid=5f458e1a-fa67-4ae6-a972-54ea42ec670e&sktid=975f013f-7f24-47e8-a7d3-abc4752bf346&skt=2021-07-28T03%3A49%3A06Z&ske=2021-08-04T03%3A54%3A06Z&sks=b&skv=2020-04-08&sv=2020-04-08&st=2021-07-28T09%3A25%3A01Z&se=2021-08-28T09%3A30%3A01Z&sr=b&sp=r&sig=fktlSslo%2F4mqunq3qjfZoj9S4aK5tpF3%2FRykp9rW19s%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/db9f4228-bcf7-7df9-527f-b7c9a032ef40?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

captainbrosset commented:
Thanks @mikehoffms, I think I addressed all of your comments. I did change the images to have red borders instead of yellow. But we might want to redo them later to have 3px borders instead of 5px (plus also to clean them up as painting the yellow regions to red left a thin yellow artefact that's not visible unless you zoom in quite a lot, so I think it's probably ok for now).

opbld32 commented:
Docs Build status updates of commit _[a8dc1d1](https://github.com/MicrosoftDocs/edge-developer/commits/a8dc1d187d80935223a7c8085644d94e82b0e980)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-edge/visual-studio/index.md](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |
[microsoft-edge/visual-studio/media/attach-to-process.PNG](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/media/attach-to-process.PNG) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |
[microsoft-edge/visual-studio/media/create-new-project.png](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/media/create-new-project.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |
[microsoft-edge/visual-studio/media/edge-counter.png](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/media/edge-counter.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |
[microsoft-edge/visual-studio/media/edge-launch.png](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/media/edge-launch.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |
[microsoft-edge/visual-studio/media/enable-script-debugging.png](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/media/enable-script-debugging.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |
[microsoft-edge/visual-studio/media/feedback-icon.png](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/media/feedback-icon.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |
[microsoft-edge/visual-studio/media/set-breakpoint.png](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/media/set-breakpoint.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |
[microsoft-edge/visual-studio/media/set-web-browser.png](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/media/set-web-browser.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |
[microsoft-edge/visual-studio/media/vs-dropdown.png](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/media/vs-dropdown.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/index?branch=pr-en-us-1360) |
[microsoft-edge/visual-studio/media/vs.msft.png](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/vs-edge/microsoft-edge/visual-studio/media/vs.msft.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio/media/vs.msft.png?branch=pr-en-us-1360) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C7%5C29%5Cdb9f4228-bcf7-7df9-527f-b7c9a032ef40%5CPullRequest%5C202107291749084920-1360%5Cworkflow_report.html?skoid=5f458e1a-fa67-4ae6-a972-54ea42ec670e&sktid=975f013f-7f24-47e8-a7d3-abc4752bf346&skt=2021-07-28T03%3A56%3A44Z&ske=2021-08-04T04%3A01%3A44Z&sks=b&skv=2020-04-08&sv=2020-04-08&st=2021-07-29T17%3A45%3A53Z&se=2021-08-29T17%3A50%3A53Z&sr=b&sp=r&sig=8RQndYPEsZPuZfJAcwQru8hJkwI1g7aNIgV9WCf31%2Bw%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/db9f4228-bcf7-7df9-527f-b7c9a032ef40?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>
